app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -270,6 +270,7 @@ def cloud_volume_types
       persister.cloud_volume_types.build(
         :type        => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolumeType",
         :ems_ref     => v.type,
+        :ems_id      => persister.storage_manager.id,
```

Comment:
I'm not sure what changed to make it necessary to set this now (I know in the past the relationship to the parent storage manager got _auto-magically_ set).

I'll have to go back and test `lasker` and `kasparov` to see if this bug is present.

Reply:
If these don't have a core persister collection which it looks like they don't then I'm guessing it was this: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/152/files#diff-e41645b87bcd5ddfe80fc3e2da3788b5a19b2da616dc10b96ea69715fbd4d344L68-L70

It looked indistinguishable from an override of a core collection to set the storage-manager instead of the cloud-manager.